src/components/designSystemExtension/DetailsFields/index.ts

```diff
@@ -102,6 +109,8 @@ class DetailsFields extends LitElement {
           </dd>
```

Comment:
I know this is in existing code but... it looks like it's supposed to close dt, not dd

Reply:
Looks like the code diff threw me off. So this is ok for now.